pallets/multisig/src/lib.rs

```diff
@@ -1111,6 +1106,17 @@ impl<T: Trait> MultiSigSubTrait<T::AccountId> for Module<T> {
     }
 
     fn is_multisig(account: &T::AccountId) -> bool {
-        <NumberOfSigners<T>>::contains_key(account)
+        <MultiSigToIdentity<T>>::contains_key(account)
+    }
+
+    /// Helper function that returns if a key is a multisig signer
+    fn is_signer(key: &T::AccountId) -> bool {
+        <KeyToMultiSig<T>>::contains_key(key)
+    }
+
+    /// Helper function that returns if a key is a multisig signer
```

Comment:
Comments needs updating.

Reply:
This function was removed as it is no longer needed